pennylane_lightning/core/src/simulators/lightning_gpu/utils/CMakeLists.txt

```diff
@@ -5,9 +5,4 @@ findCUDATK(lightning_compile_options)
 
 add_library(${PL_BACKEND}_utils INTERFACE)
 target_include_directories(${PL_BACKEND}_utils INTERFACE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>)
-target_link_libraries(${PL_BACKEND}_utils INTERFACE CUDA::cudart CUDA::cublas CUDA::cusparse lightning_compile_options)
-
-if (BUILD_TESTS)
-    enable_testing()
-    add_subdirectory("tests")
-endif()
+target_link_libraries(${PL_BACKEND}_utils INTERFACE CUDA::cudart CUDA::cublas CUDA::cusparse lightning_compile_options)
```

Comment:
```suggestion
target_link_libraries(${PL_BACKEND}_utils INTERFACE CUDA::cudart CUDA::cublas CUDA::cusparse lightning_compile_options)

```

Reply:
@multiphaseCFD will you accept these suggestions? The style here is by adding an empty line to remove this :no_entry: signs and keep the consistency.

Reply:
@maliasadi Nice catch! I thought I added the blank line...